config/elasticsearch.php

```diff
@@ -2,6 +2,8 @@
 
 return [
     'host' => env('ELASTICSEARCH_HOST', 'localhost:9200'),
+    'username' => env('ELASTICSEARCH_USERNAME', null),
+    'password' => env('ELASTICSEARCH_PASSWORD', null),
```

Comment:
`null` is default, so you don't have to provide it as fallback.

Reply:
yeah, i know, but in this case i prefer it being explicit. your call though :)

Reply:
It is uncommon to explicitly provide `null` as fallback and it even gets removed in the scaffold itself, see https://github.com/laravel/laravel/pull/5791 for example.